weave-js/src/components/Button/Button.tsx

```diff
@@ -124,7 +124,7 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
               'bg-transparent': isGhost,
               'text-moon-600 dark:text-moon-400': isGhost,
               'hover:bg-oblivion/[0.07] hover:text-moon-800 dark:hover:bg-moonbeam/[0.09] dark:hover:text-moon-200':
-                isGhost,
+                isGhost && !active,
```

Comment:
minor style fix. currently if you hover an active ghost button, it'll go into the grey hover state instead staying in active state. this change ensures active button stays in active state.

https://github.com/user-attachments/assets/74ab0a2c-6874-43fb-b81e-63096e240341